.github/workflows/pypi.yml

```diff
@@ -0,0 +1,102 @@
+name: PyPI
+on:
+  push:
+    branches:
+      - master
+      - auto-release
+  pull_request:
+    branches: [master]
+  release:
+    types: [published]
```

Comment:
Generally with Dask projects we do not use the GitHub releases feature. We just create new git tags and push them.

So I'm not sure this would ever be triggered. We should instead trigger on tags.

Reply:
Luckily there's a way to [trigger a GitHub action workflow run on tags, on either push or pull request](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestbranchestags). What versions are we thinking?

Reply:
All. You should be able to do something like 

```yaml
on:
  push:
    tags:        
      - *
```